examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+power_axis = nddata(read_array['power'],[-1],['time'])
+power_axis.setaxis('time',relative_time)
+power_axis.name('power')
+fl.next('power axis W')
+coupler_atten = 22
+power_axis.data = (10**((power_axis.data+coupler_atten)/10+3))/1e6 #convert to W
+fl.plot(power_axis,'.')
+#}}}
+#{{{IR processing
+T1_list = []
+power_list = []
+start_times = []
+stop_times = []
+errors=[]
+for nodename, postproc, signal_pathway, clock_correction in [
+        ('FIR_noPower','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_27dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_30dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_32dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ('FIR_33dBm','spincore_IR_v1',{'ph1':0,'ph2':1},False),
+        ]:
+    IR = find_file(filename,exp_type=file_location,expno=nodename,
+            postproc=postproc,lookup=lookup_table)
+    times = IR.C.getaxis('indirect').copy()
+    IR.mean('indirect')
+    fl.next('Raw IR')
+    fl.image(IR.C.setaxis('vd','#').set_units('vd','scan #'))
+    IR['ph2',0]['ph1',0]['t2':0]=0 #kill axial noise
+    IR.ift('t2')
+    IR.ift(['ph1','ph2'])
+    t_start = t_range[-1]/4
+    t_start *= 3
+    rx_offset_corr = IR['t2':(t_start,None)]
+    rx_offset_corr = rx_offset_corr.data.mean()
+    IR -= rx_offset_corr
+    IR.ft('t2')
+    IR.ft(['ph1','ph2'])
+    zero_crossing = abs(select_pathway(IR['t2':IR_f_slice],signal_pathway)).C.sum('t2').argmin('vd',raw_index=True).item()
+    IR=IR['t2':IR_f_slice]
+    if 'nScans' in IR.dimlabels:
+        IR.mean('nScans')
+    IR.ift('t2')
+    #{{{clock correction
+    if clock_correction:
+        clock_corr = nddata(np.linspace(-3,3,2500),'clock_corr')
+        IR.ft('t2')
+        if fl is not None:
+            fl.next('before clock correction')
+            fl.image(IR.C.setaxis('vd','#').set_units('vd','scan #'))
+        s_clock=IR['ph1',1]['ph2',0].sum('t2')
+        IR.ift(['ph1','ph2'])
+        min_index = abs(s_clock).argmin('vd',raw_index=True).item()
+        s_clock *= np.exp(-1j*clock_corr*IR.fromaxis('vd'))
+        s_clock['vd',:min_index+1] *=-1
+        s_clock.sum('vd').run(abs)
+        if fl is not None:
+            fl.next('clock correction')
+            fl.plot(s_clock,'.',alpha=0.7)
+        clock_corr = s_clock.argmax('clock_corr').item()
+        plt.axvline(x=clock_corr, alpha=0.5, color='r')
+        IR *= np.exp(-1j*clock_corr*IR.fromaxis('vd'))
+        IR.ft(['ph1','ph2'])
+        if fl is not None:
+            fl.next('after auto-clock correction')
+            fl.image(IR.C.setaxis('vd','#'))
+        IR.ift('t2')
+    #}}}
+    #{{{phasing
+    IR.ft('t2')
+    IR.ift('t2')
+    best_shift = hermitian_function_test(select_pathway(IR.C.mean('vd'),
+        signal_pathway),aliasing_slop=0)
+    best_shift = 0.0035 #this data is bad (bad tuning) so I hard set the shift
+    IR.setaxis('t2',lambda x: x-best_shift).register_axis({'t2':0})
+    IR.ft('t2')
+    IR.ift('t2')
```

Comment:
just do this at the very beginning.  we don't need to see the multiple steps here.